tools/autograd/gen_variable_type.py

```diff
@@ -369,6 +369,9 @@
     "linalg_lu_solve",
     "_linalg_slogdet",
     "_linalg_solve_ex",
+    "silu",
+    "silu_backward",
+    "cpu",
```

Comment:
what does this mean?

Reply:
I thought I needed to add `cpu` to that list because you said I needed to add the support for the CPU kernels.

Reply:
I removed it

https://github.com/pytorch/pytorch/pull/89649/commits/d7649c6961ece57847dc25ab018d649e8408e6be